tests/integration/test_harmony.py

```diff
@@ -10,13 +10,9 @@ def test_harmony(adata_paul15_template):
     )
 
     # check NMI after clustering
-    res_max, score_max, _ = scib.cl.opt_louvain(
-        adata,
-        label_key="celltype",
-        cluster_key="cluster",
-        plot=False,
-        inplace=True,
+    res_max, score_max, _ = scib.cl.cluster_optimal_resolution(
+        adata, label_key="celltype", cluster_key="cluster", return_all=True
```

Comment:
order of parameters is different from order in `conftest.py` could you make this consistent?

Reply:
yes, leiden is the default for the `cluster_optimal_resolution` function.